tests/integration/sentinel/transitions/registration.spec.js

```diff
@@ -98,7 +97,225 @@ describe('registration', () => {
   after(() => utils.revertDb([], true));
   afterEach(() => utils.revertDb(getIds(contacts), true));
 
-  it('should be skipped when transition is disabled', () => {
+  it.only('should add valid phone to patient doc', () => {
```

Comment:
Please revert the `.only` so all tests run in CI.

Reply:
Thanks. Done.